src/Domain/Services/JetpackWooCommerceAnalytics.php

```diff
@@ -334,4 +340,51 @@ private function get_nested_blocks( $blocks, $exclude = array() ) {
 		}
 		return $additional_blocks;
 	}
+
+	/**
+	 * Track local pickup settings changes via Store API
+	 *
+	 * @param bool              $served Whether the request has already been served.
+	 * @param \WP_REST_Response $result The response object.
+	 * @param \WP_REST_Request  $request The request object.
+	 * @return bool
+	 */
+	public function track_local_pickup( $served, $result, $request ) {
+		if ( '/wp/v2/settings' !== $request->get_route() ) {
+			return $served;
+		}
+		// Param name here comes from the show_in_rest['name'] value when registering the setting.
+		if ( ! $request->get_param( 'pickup_location_settings' ) && ! $request->get_param( 'pickup_locations' ) ) {
+			return $served;
+		}
+
+		if ( ! $this->is_compatible ) {
+			return $served;
+		}
+
+		$event_name = 'wcadmin_local_pickup_save_changes';
```

Comment:
This actually tracks as `wcadmin_wcadmin_local_pickup_save_changes` should we remove the prefix here?
```suggestion
		$event_name = 'local_pickup_save_changes';
```